css-canvas-medical/styles.css

```diff
@@ -0,0 +1,248 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 48px;
+}
+
+.navbar,
+.navlinks {
+  font-family: Avenir, sans-serif;
+}
+
+.navlinks {
+  display: inline;
+  margin-right: 30px;
+  font-size: 11px;
+}
+
+.navlinks > a {
+  color: #afadad;
+  text-decoration: none;
+}
+
+.nav-logo {
+  width: 100px;
+  margin-left: 55px;
+}
+
+.container {
+  width: 900px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+  margin: auto;
+}
+
+.column-half {
+  width: 50%;
+  margin: auto;
+}
+
+.column-fourth {
+  width: 25%;
+  margin: auto;
+}
```

Comment:
Instead of applying `margin: auto` to your column classes, you should make a separate class with that styling and apply it to only the column elements that require it.

Reply:
AWESOME! Cleaner column declarations!